metadata-jobs/mce-consumer-job/src/main/java/com/linkedin/metadata/restli/EbeanServerConfig.java

```diff
@@ -0,0 +1,74 @@
+package com.linkedin.metadata.restli;
+
+import io.ebean.datasource.DataSourceConfig;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.linkedin.gms.factory.common.LocalEbeanServerConfigFactory.getListenerToTrackCounts;
+
+@Configuration
+public class EbeanServerConfig {
+    @Value("${ebean.username}")
+    private String ebeanDatasourceUsername;
+
+    @Value("${ebean.password}")
+    private String ebeanDatasourcePassword;
+
+    @Value("${ebean.driver}")
+    private String ebeanDatasourceDriver;
+
+    @Value("${ebean.minConnections:1}")
+    private Integer ebeanMinConnections;
+
+    @Value("${ebean.maxInactiveTimeSeconds:120}")
+    private Integer ebeanMaxInactiveTimeSecs;
+
+    @Value("${ebean.maxAgeMinutes:120}")
+    private Integer ebeanMaxAgeMinutes;
+
+    @Value("${ebean.leakTimeMinutes:15}")
+    private Integer ebeanLeakTimeMinutes;
+
+    @Value("${ebean.waitTimeoutMillis:1000}")
+    private Integer ebeanWaitTimeoutMillis;
+
+    @Value("${ebean.autoCreateDdl:false}")
+    private Boolean ebeanAutoCreate;
+
+    @Value("${ebean.postgresUseIamAuth:false}")
+    private Boolean postgresUseIamAuth;
+
+
+    @Bean("ebeanDataSourceConfig")
+    @Primary
+    public DataSourceConfig buildDataSourceConfig(
+            @Value("${ebean.url}") String dataSourceUrl,
+            @Qualifier("parseqEngineThreads") int ebeanMaxConnections
```

Comment:
I'm assuming it's intentional that we are matching parseqThread # with ebeanMaxConnection number? Feel we may want to segment these configs out?

Reply:
This is intentional. A single thread should only need 1 connection. Running more connections than needed merely cycles through the connections in a larger pool to make sure they are all alive, however only 1 connection per thread is ever actively used. This is only for restli in mce-consumer for now so its impact doesn't extend to GMS in general.